great_expectations/rule_based_profiler/parameter_container.py

```diff
@@ -13,41 +13,48 @@
     alphas,
     nums,
 )
+from typing_extensions import Final
 
 import great_expectations.exceptions as ge_exceptions
 from great_expectations.core.domain import Domain
 from great_expectations.core.util import convert_to_json_serializable
 from great_expectations.types import SerializableDictDot, SerializableDotDict
 
-FULLY_QUALIFIED_PARAMETER_NAME_DELIMITER_CHARACTER: str = "$"
-
-FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER: str = "."
-
-DOMAIN_KWARGS_PARAMETER_NAME: str = "domain_kwargs"
-DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME: str = f"{FULLY_QUALIFIED_PARAMETER_NAME_DELIMITER_CHARACTER}domain{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{DOMAIN_KWARGS_PARAMETER_NAME}"
-
-PARAMETER_NAME_ROOT_FOR_VARIABLES: str = "variables"
-VARIABLES_PREFIX: str = f"{FULLY_QUALIFIED_PARAMETER_NAME_DELIMITER_CHARACTER}{PARAMETER_NAME_ROOT_FOR_VARIABLES}"
-VARIABLES_KEY: str = (
-    f"{VARIABLES_PREFIX}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}"
-)
-
-PARAMETER_NAME_ROOT_FOR_PARAMETERS: str = "parameter"
-PARAMETER_PREFIX: str = f"{FULLY_QUALIFIED_PARAMETER_NAME_DELIMITER_CHARACTER}{PARAMETER_NAME_ROOT_FOR_PARAMETERS}"
-PARAMETER_KEY: str = (
-    f"{PARAMETER_PREFIX}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}"
-)
-
-RAW_SUFFIX: str = "raw"
-RAW_PARAMETER_KEY: str = (
-    f"{PARAMETER_KEY}{RAW_SUFFIX}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}"
-)
-
-FULLY_QUALIFIED_PARAMETER_NAME_ATTRIBUTED_VALUE_KEY: str = "attributed_value"
-FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY: str = "details"
-FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY: str = "value"
-
-RESERVED_TERMINAL_LITERALS: Set[str] = {
+FULLY_QUALIFIED_PARAMETER_NAME_DELIMITER_CHARACTER: Final[str] = "$"
```

Comment:
@Kilo59 Again, love the use of `Final` -- does it need to be imported/defined?  Thanks.

Reply:
It is imported on line `16`